README.md

```diff
@@ -96,21 +96,9 @@ Bee-Queue is like a bee because it:
 
 # Benchmarks
 
-![benchmark chart](https://raw.githubusercontent.com/bee-queue/bee-queue/master/benchmark/benchmark-chart.png)
+![benchmark chart](./benchmark/results-2017-08-12/resultschart.png)
```

Comment:
This will make the image not load on npmjs.com - what's the benefit to doing this?